.github/workflows/run-standard-tests.yml

```diff
@@ -0,0 +1,143 @@
+---
+name: Run Standard Tests
+# Run minimal tests for Khiops, Khiops Coclustering and KNITransfer
+# Tests are lauched on Linux, Windows and macOS ; in serial and in parallel (3 procs)
+# On release, it runs Standard test for binary. Plus MultiTables/SpliceJunction for KNITransfer
+# On debug, it runs only Standard/Iris for Coclustering and KNITransfer and Standard/IrisLight for Khiops
+# In case of errors, test results are available as artifacts for 5 days
+# It is automatically triggered on PR when sources files are modified (files in src directory)
+on:
+  workflow_dispatch:
+  pull_request:
+    paths: [src/**]
```

Comment:
Peut-être que cette branche doit être mergée avant ?
En tout cas je ne souhaite pas que ce soit lancé lors de chaque push

Reply:
I think it is because there are no files in `src` that changed in this PR, so it is working :-). Ok


Just for info: After some investigation `pull_request` is better than `push` as it will be triggered only by pushes to a PR and not *any* push (for example, when merging to `dev` there is a `push` event).